modules/FiniteSetsExt.tla

```diff
@@ -54,4 +54,32 @@ kSubset(k, S) ==
    (*************************************************************************)
    { s \in SUBSET S : Cardinality(s) = k }
 
+-----------------------------------------------------------------------------
+
+(***************************************************************************)
+(* We define Max(S) and Min(S) to be the maximum and minimum,              *)
+(* respectively, of a finite, non-empty set S of integers.                 *)
+(***************************************************************************)
+Max(S) == CHOOSE x \in S : \A y \in S : x >= y
+Min(S) == CHOOSE x \in S : \A y \in S : x =< y
+
+-----------------------------------------------------------------------------
+
+(***************************************************************************) 
+(* If Sets is a set of (non-empty) sets then Choice(Sets) is the set of    *)
+(* all choice functions over Sets, that is, functions that associate some  *)
+(* with every set in Sets.                                                 *) 
+(***************************************************************************) 
```

Comment:
I've started to add simple examples to operator comments:

https://github.com/tlaplus/CommunityModules/blob/2ebcbd910bcf5a2cc7f62cdb3a11c9e2f2d15641/modules/FiniteSetsExt.tla#L51-L55

I hope this is useful in cases where ECE is not available (e.g. reading documentation on the web).